.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
```

Comment:
@nicolas-grekas having the bridge storing `.phpunit` inside `vendor/symfony/phpunit-bridge/bin` by default makes it hard to cache this folder on Travis (caching the folder inside the vendor folder won't work fine as it will be overwritten when Composer installs `symfony/phpunit-bridge`, and so it requires overwriting the location). Can't we find an easier location ?